sql/plan/drop_view.go

```diff
@@ -0,0 +1,148 @@
+package plan
+
+import (
+	"github.com/src-d/go-mysql-server/sql"
+	errors "gopkg.in/src-d/go-errors.v1"
+)
+
+var errDropViewChild = errors.NewKind("any child of DropView must be of type SingleDropView")
+
+type SingleDropView struct {
+	database sql.Database
+	viewName string
+}
+
+// NewSingleDropView creates a SingleDropView.
+func NewSingleDropView(
+	database sql.Database,
+	viewName string,
+) *SingleDropView {
+	return &SingleDropView{database, viewName}
+}
+
+// Children implements the Node interface. It always returns nil.
+func (dv *SingleDropView) Children() []sql.Node {
+	return nil
+}
+
+// Resolved implements the Node interface. This node is resolved if and only if
+// its database is resolved.
+func (dv *SingleDropView) Resolved() bool {
+	_, ok := dv.database.(sql.UnresolvedDatabase)
+	return !ok
+}
+
+// RowIter implements the Node interface. It always returns an empty iterator.
+func (dv *SingleDropView) RowIter(ctx *sql.Context) (sql.RowIter, error) {
+	return sql.RowsToRowIter(), nil
+}
+
+// Schema implements the Node interface. It always returns nil.
+func (dv *SingleDropView) Schema() sql.Schema { return nil }
+
+// String implements the fmt.Stringer interface, using sql.TreePrinter to
+// generate the string.
+func (dv *SingleDropView) String() string {
+	pr := sql.NewTreePrinter()
+	_ = pr.WriteNode("SingleDropView(%s.%s)", dv.database.Name(), dv.viewName)
+
+	return pr.String()
+}
+
+// WithChildren implements the Node interface. It only succeeds if the length
+// of the specified children equals 0.
+func (dv *SingleDropView) WithChildren(children ...sql.Node) (sql.Node, error) {
+	if len(children) != 0 {
+		return nil, sql.ErrInvalidChildrenNumber.New(dv, len(children), 0)
+	}
+
+	return dv, nil
+}
+
+// Database implements the Databaser interfacee. It returns the node's database.
```

Comment:
Typo (interfacee).

Reply:
Thank you! Done: https://github.com/src-d/go-mysql-server/pull/862/commits/03294864a13d85a19179de4ae51b5fcb2025d4ff